src/logic/safe/transactions/pendingTxMonitor.ts

```diff
@@ -2,18 +2,31 @@ import { backOff } from 'exponential-backoff'
 
 import { NOTIFICATIONS } from 'src/logic/notifications'
 import enqueueSnackbar from 'src/logic/notifications/store/actions/enqueueSnackbar'
-import { getWeb3ReadOnly } from 'src/logic/wallets/getWeb3'
+import { getWeb3 } from 'src/logic/wallets/getWeb3'
 import { store } from 'src/store'
 import { removePendingTransaction } from 'src/logic/safe/store/actions/pendingTransactions'
 import { pendingTxIdsByChain } from 'src/logic/safe/store/selectors/pendingTransactions'
 
+const MAX_ATTEMPTS = 6
```

Comment:
Over how many minutes is it?

Reply:
This is number of attempts, not minutes.

It exponentially increases by a factor of 2, starting at 10s.

Reply:
Yes, so how much time does it take in total?

Reply:
I've moved these into constants to make it clearer.

Reply:
6.5 min happens to be an epoch duration on Eth2 :)